api/python/stat_value.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Place Statistics - Single Value
+nav_order: 11
+parent: Python
+grand_parent: API
+---
+
+# Get Statistics Value for a Place.
+
+## `get_stat_value(place, stat_var, date=None, measurement_method=None,observation_period=None, unit=None, scaling_factor=None)`
+
+Returns a value for `place` based on the
+[`stat_var`](https://datacommons.org/browser/StatisticalVariable), with optional
+filter parameters.
+
+See the [full list of StatisticalVariables](/statistical_variables.html).
+
+**Arguments**
+
+* `place (str)`: The `dcid` of the
+  [`Place`](https://datacommons.org/browser/Place) to query for.
+
+* `stats_var (str)`: The `dcid` of the
+  [`StatisticalVariable`](https://datacommons.org/browser/StatisticalVariable).
+
+* `date (str)`: (Optional) The preferred date of the observation in ISO-8601 format, e.g. "YYYY", "YYYY-MM" or "YYYY-MM-DD". If not specified, returns the latest observation.
+
+* `measurement_method (str)`: (Optional) The `dcid` of the preferred `measurementMethod` for the `stat_var`.
+
+* `observation_period (str)`: (Optional) The preferred `observationPeriod` for the `stat_var`.
+
+* `unit (str)`: (Optional) The `dcid` of the preferred `unit` for the `stat_var`.
+
+* `scaling_factor (str)`: (Optional) The preferred `scalingFactor` for the `stat_var`.
+
+**Returns**
+
+ A `float` value of the `stat_var` for `place`, filtered by the optional parameters. If the optional parameters are specified, but are unavailable for the `stat_var`, then ... is returned.
```

Comment:
please fill the ___ in, and an example of when it's overspecified (with and without possible results).

Reply:
confirmed with tjann,  if there is no value found, an ValueError will be thrown. Updated this part.